common/app/views/fragments/email/signup/footer/emailSignUpFooter.scala.html

```diff
@@ -0,0 +1,44 @@
+@(listName: String)(implicit request: RequestHeader)
+
+@import common.LinkTo
+@formId = @{"footer-email-sub-form"}
+@inputId = @{"footer-email-sub-input" }
+@dummyInputId = @{"footer-email-sub-input-name" }
+
+@wrapperClass = @{ "email-sub email-sub--footer" }
+@formClass = @{ "email-sub__form email-sub__form--footer" }
+@headerClass = @{"email-sub__header email-sub__header--footer" }
+
+@formHeading = @{"Sign up for the Guardian Today email"}
+@formDescription = @{"All the day's headlines and highlights from the Guardian, direct to you every morning"}
+
+
+@form = {
+    <form action="@LinkTo("/email/footer")" method="post" id="@formId" class="@formClass" data-email-form-type="footer" data-email-list-name="@listName">
```

Comment:
it might be easier to have a separate route for `/email/footer` that's higher precedence than `/email/:listName` so that you can deal with them separately

Reply:
See previous about this